### ISSUE TITLE
Utilities Device adapter, DA-TTL-State-Device voltage property error when in System-Startup

### ISSUE THREAD

**Issue by nicost** [micro-manager/mmCoreAndDevices#504]
When using the DA-TTL-State-Device property "TTL Voltage" in the System-Startup group (which is kind of where it belongs), a runtime error happens.

**Comment by nicost** [comment]
The error can be reproduced with the Demo Config, by adding two demo DAs, and the DA-TTL-State-Device.  I do add the inverted logic property to the System-Startup group as well.  
Corelog:
```
2024-10-16T10:12:18.428725 tid29880 [IFO,dev:D-DA] Voltage set to 3.3
2024-10-16T10:12:18.428726 tid29880 [IFO,dev:D-DA2] Voltage set to 3.3
2024-10-16T10:12:18.428731 tid29880 [ERR,Core:dev:DA TTL State Device] Cannot set property "TTL Voltage" to "3.30" [ Error in device "DA TTL State Device": Invalid property value: TTL Voltage (3) ]
2024-10-16T10:12:18.428748 tid29880 [ERR,Core:dev:DA TTL State Device] Cannot set property "TTL Voltage" to "3.30" [ Error in device "DA TTL State Device": Invalid property value: TTL Voltage (3) ]
2024-10-16T10:12:18.428753 tid29880 [ERR,Core] Error occurred in device DA TTL State Device: Cannot set property "TTL Voltage" to "3.30" [ Error in device "DA TTL State Device": Invalid property value: TTL Voltage (3) ]
```

**Comment by nicost** [comment]
Turns out that the function:
`bool MM::Property::IsAllowed(const char* value) const`
in Property.cpp (line 54) returns false, even though the map 'values_' contains the string "3.3" and the value fed into the function is "3.3".  I do not understand at all why this is, especially since setting the property in other settings (such as in the device/property browser) works fine.

**Comment by nicost** [comment]
OK, dyslexia.  It looks like the value stored in the config file is "3.30", whereas the value accepted is "3.3".  I guess that something in Micro-Manager interprets the string "3.3" as a float and adds a second digit.  This is certainly unexpected behavior for a String property.  There are many way to work around this (I'll probably end up adding "v" to the voltage value strings), but we may want to think about ways to avoid this kind of weirdness.

**Comment by nicost** [comment]
Turns out this behavior originates from the Device/Property Browser.  It lists the voltages as 3.30 and 5.00 instead of 3.3 and 5.0.  Interestingly, this does not lead to unexpected behaviors in the Device/Property Browser itself, only when saving settings in the configuration file.  I chased this further down, and the 0 is added in the function `core.getPropertyFromCache(deviceLabel, propName)`.  I am confused where this happens, somewhere deep in the core, so I will work around it for now.  We may need another issue for this.
